src/components/Pages/index.js

```diff
@@ -230,7 +223,7 @@ const VmCreatePageConnected = connect(
     canUserCreateVMs: canUserUseAnyClusters(state.clusters) && state.clusters.size > 0,
   }),
   (dispatch) => ({
-    getAvailableCDImages: () => dispatch(getIsoStorageDomains()),
+    refreshPage: () => dispatch(refreshDialogPage({})),
```

Comment:
Oops!  Should be `refreshDialogPage`.  "Create Virtual Machine" bombs immediately.